src/features/home/pages/Home.tsx

```diff
@@ -21,12 +21,12 @@ type Props = {
 export const Home: FunctionComponent<Props> = function ({ navigation }) {
   const { data: modules = [] } = useHomepageModules()
 
-  const goToLoginPage = useCallback((): void => {
-    navigation.navigate('Login')
-  }, [])
+  const goToLoginPage = useCallback((): void => navigation.navigate('Login'), [])
+  const goToComponentsPage = useCallback((): void => navigation.navigate('AppComponents'), [])
 
   return (
     <ScrollView>
+      <Button title={_(t`Components`)} onPress={goToComponentsPage} />
```

Comment:
```suggestion
      <Button title={_(t`Composants`)} onPress={goToComponentsPage} />
```
Et lances `yarn translations:extract` :smile: